src/gtk/toplevel.cpp

```diff
@@ -1163,6 +1165,12 @@ bool wxTopLevelWindowGTK::Show( bool show )
     bool change = base_type::Show(show);
 
 #ifdef __WXGTK3__
+    if (change && show)
+    {
+        // We may need to redo it after showing the window.
+        GTKUpdateClientSizeIfNecessary();
```

Comment:
I think this is redundant. Won't the `map-event` handler have already done this?

Reply:
I don't quite understand when is `GTKDoAfterShow()` called from `gtk_frame_map_callback()`, i.e. how to reproduce [#9909](https://trac.wxwidgets.org/ticket/9909) referenced there, but in my test this doesn't happen, i.e. the window is always already considered to be shown. I suspect this might not happen at all any longer, but I'm definitely not 100% sure about this... And calling `GTKUpdateClientSizeIfNecessary()` twice, which is the worst that can happen here, should do no harm, AFAICS.